lib/truffle/java/core_ext.rb

```diff
@@ -0,0 +1,32 @@
+###### BEGIN LICENSE BLOCK ######
+# Version: EPL 1.0/GPL 2.0/LGPL 2.1
+#
+# The contents of this file are subject to the Common Public
+# License Version 1.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of
+# the License at http://www.eclipse.org/legal/cpl-v10.html
+#
+# Software distributed under the License is distributed on an "AS
+# IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# rights and limitations under the License.
+#
+# Alternatively, the contents of this file may be used under the terms of
+# either of the GNU General Public License Version 2 or later (the "GPL"),
+# or the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
+# in which case the provisions of the GPL or the LGPL are applicable instead
+# of those above. If you wish to allow use of your version of this file only
+# under the terms of either the GPL or the LGPL, and not to allow others to
+# use your version of this file under the terms of the EPL, indicate your
+# decision by deleting the provisions above and replace them with the notice
+# and other provisions required by the GPL or the LGPL. If you do not delete
+# the provisions above, a recipient may use your version of this file under
+# the terms of any one of the EPL, the GPL or the LGPL.
+###### END LICENSE BLOCK ######
+
+# Extensions to Ruby classes
+
+# These are loads so they don't pollute LOADED_FEATURES
+load File.expand_path('../core_ext/module.rb', __FILE__)
+load File.expand_path('../core_ext/object.rb', __FILE__)
+load File.expand_path('../core_ext/kernel.rb', __FILE__)
```

Comment:
I'm not sure what the concern was in JRuby, but they very specifically used load rather than require relative.